components/script/task_source/user_interaction.rs

```diff
@@ -40,7 +40,8 @@ impl TaskSource for UserInteractionTaskSource {
         let msg = MainThreadScriptMsg::Common(CommonScriptMsg::Task(
             ScriptThreadEventCategory::InputEvent,
             Box::new(canceller.wrap_task(task)),
-            Some(self.1)
+            Some(self.1),
+            TaskSourceName::UserInteraction,
```

Comment:
For all the changes inside implementation of `TaskSource`, like this one, I think you can use the associated const directly, doing something like `UserInteractionTaskSource::NAME`